docs/account.md

```diff
@@ -2,6 +2,8 @@
 
 [<-Back-to:Auth](database-auth.md)
 
+Read in: [English :gb:](account) [Spanish :es:](es/account)
```

Comment:
No. I dont think this is a good change. As far as I understand it, if you change on the home page all links would be in that locale. So I do not like this change. Please revert it

Reply:
I don't know, it depends perhaps to allow the user to change the language of the pages more easily for example:
You are on the English wiki inside the acount table in the auth database https://www.azerothcore.org/wiki/account and you want to read that same table in Spanish, it is easier for the user to give it directly there is the same to Spanish than do not change the language in the sidebar look for db-auth and there is inside the file that contains the tables look for the acoount table to read it in Spanish.

If you think that example is not correct, tell me and I will remove the language change

Reply:
I dont like it. Maybe it is possible to make the sidebar language thing to also force the current webpage to go to en/es

Reply:
all ready @Kitzunu https://github.com/azerothcore/wiki/pull/602/commits/305e067fa90ea1ae398d5b6740617e2acfb851b9